packages/Search/examples/viz/tree_visualization.cpp

```diff
@@ -0,0 +1,157 @@
+/****************************************************************************
+ * Copyright (c) 2012-2019 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <DTK_DetailsTreeVisualization.hpp>
+#include <DTK_LinearBVH.hpp>
+
+#include <Kokkos_Core.hpp>
+#include <Kokkos_DefaultNode.hpp>
+
+#include <boost/program_options.hpp>
+
+#include <algorithm>
+#include <fstream>
+#include <random>
+
+#include <point_clouds.hpp>
+
+template <typename View>
+void printPointCloud( View points, std::ostream &os )
+{
+    auto const n = points.extent_int( 0 );
+    for ( int i = 0; i < n; ++i )
+        os << "\\node[leaf] at (" << points( i )[0] << "," << points( i )[1]
+           << ") {\\textbullet};\n";
+}
+
+template <typename TreeType>
+void viz( std::string const &prefix, std ::string const &infile,
```

Comment:
there is weird blank space between `std` and `::string`

Reply:
That's weird.  `clang-format` didn't catch it?!

Reply:
Fixed.  Still no idea why `clang-format` does not fix it.